target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -26,6 +26,10 @@ case "$FIRMWARE" in
 		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		;;
+	asus,rt-acrh17)
+		CI_UBIPART=UBI_DEV
```

Comment:
For the AC58U I wanted to get rid of the UBI_DEV partition at some point. In theory if you rename the UBI_DEV to ubi and remove all the CI_UBIPART and extra bootargs it should still work as intended.

Reply:
Tested on both AC42U and AC58U.